channels/fast-4.3.yaml

```diff
@@ -30,3 +30,6 @@ versions:
 # No 4.3.11 because of build issues: https://bugzilla.redhat.com/show_bug.cgi?id=1823378
 - 4.3.12
 - 4.3.13
+
+# 4.3.14 Errata: https://errata.devel.redhat.com/advisory/53642
+- 4.3.14
```

Comment:
should also be going into the 4.4 channels, like [this][1].

[1]: https://github.com/openshift/cincinnati-graph-data/blame/master/channels/fast-4.4.yaml#L4